zgrab_schema.py

```diff
@@ -202,7 +202,8 @@
     }),
     "session_ticket":SubRecord({
         "value":Binary(),
-        "length":Integer()
+        "length":Integer(),
```

Comment:
It's like how we store length for every other binary thing. It's difficult
to compute this in place and varies.

On Tue, Jan 5, 2016 at 6:24 PM David Adrian <notifications@github.com>
wrote:

> In zgrab_schema.py
> <https://github.com/zmap/zgrab/pull/106#discussion_r48921829>:
>
> > @@ -202,7 +202,8 @@
> >      }),
> >      "session_ticket":SubRecord({
> >          "value":Binary(),
> > -        "length":Integer()
> > +        "length":Integer(),
>
> What's the point of having length? Length is implicitly determined by the
> base64 value stored in the value field in JSON.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zmap/zgrab/pull/106/files#r48921829>.
>